src/main/java/frc/robot/Robot.java

```diff
@@ -24,6 +25,8 @@
 
   private RobotContainer m_robotContainer;
 
+  private char positionControlColor;
```

Comment:
This is probably not the right place for this.

A good principle of object-oriented programming is to try to reduce [coupling](https://en.wikipedia.org/wiki/Coupling_(computer_programming)) between the different parts of the code. This means that every class should know as little as possible about the other classes.

In regards to the requested color wheel position, that information is not needed by every part of the robot. For example, neither the drive nor the shooter care what the position of the color wheel should be. The only time we need to know the desired position of the color wheel is when we're actually setting the color wheel. So this logic should be part of the (as of yet unimplemented) `SetColorWheel` command, not part of the entire robot.